### PR TITLE
Add IE versions for CSS properties

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Part of #3801.  This provides 11 new CSS properties that have been manually tested using LambdaTest and virtual machines.

1. css.properties.height.stretch - false
2. css.properties.justify-content.flex_context.left_right - false
3. css.properties.letter-spacing.svg - 10
4. css.properties.outline-style.auto - false
5. css.properties.overflow-wrap.anywhere - false
6. css.properties.text-align-last - 5.5
7. css.properties.transition-timing-function.jump - false
8. css.properties.visibility.collapse - 10
9. css.properties.width.animatable - 11
10. css.properties.width.stretch - false
11. css.properties.width.fill - false

### CODE REVIEW

File: css/properties/justify-content.json [hunk 1]
```diff
@@ -255,8 +255,7 @@
                   "version_added": "52"
                 },
                 "ie": {
-                  "version_added": false,
-                  "notes": "This value is recognized, but has no effect."
```
Comment (Elchi3) [inline css/properties/justify-content.json]: In https://github.com/mdn/browser-compat-data/pull/3904 we agreed to leave this note in, but mark it as false (which is already is). So, please undo the note removal.

Comment (vinyldarkscratch) [reply css/properties/justify-content.json]: Oh we did?  Note returning then!

Comment (Elchi3) [reply css/properties/justify-content.json]: I think you've added it back at the wrong place?

Comment (vinyldarkscratch) [reply css/properties/justify-content.json]: I think I did too, whoops -- I'm confused as to why it didn't show up in the diff, but fixed it now!

**Pull request merged**